_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -0,0 +1,483 @@
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+#### Passed Example 4
+
+This [HTML document][] has two [keyboard events][keyboard event] [dispatched][] to an [event target][] with the attribute `key` being a [printable character][] and the method `getModifierState` returning `false`, and exists an [instrument][] to **remap** the [keyboard event][] so that it requires that the method `getModifierState` returns `true`. In this example, the same [instrument][] is used to **remap** both [keyboard events][keyboard event]
+
+```html
+<html>
+  <head>
+    <title>Passed Example 4</title>
+    <script src="/test-assets/ffbc54/shortcut.js"></script>
+  </head>
+  <body onload="registerShortcut({id: 'firstShortcut', shortcutKey: '+'}); registerShortcut({id: 'secondShortcut', shortcutKey: 'a'}); activateShortcuts();">
+    <label for="target">Add to list (press "+" or "a" to add):</label>
+    <input type="text" id="target" />
+    <div>
+      <div>Remap shortcut</div>
+      <div>
+        <label>
+          <input id="remap" type="checkbox" onclick="toggleModifier('firstShortcut', this.checked); toggleModifier('secondShortcut', this.checked);">
+          Use "ctrl" key together with the "+" or "a" key
+        </label>
+      </div>
+    </div>
+    <br />
+    <div>
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+#### Passed Example 5
+
+This [HTML document][] has a [keyboard event][] [dispatched][] to an [event target][] with the attribute `key` being a [printable character][] and the method `getModifierState` returning `false`, but the [event is blocked][blocked event] when the [event target][] doesn't have [focus][].
+
+```html
+<html>
+  <head>
+    <title>Passed Example 5</title>
+    <script src="/test-assets/ffbc54/shortcut.js"></script>
+  </head>
+
+  <body onload="registerShortcut({shortcutKey: '+', focusOnly: true}); activateShortcuts();">
+    <label for="target">Add to list (press "+" to add):</label>
+    <input type="text" id="target" />
+    <br />
+    <div>
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+#### Passed Example 6
+
+This [HTML document][] has a [keyboard event][] [dispatched][] to an [event target][] with the attribute `key` being a [printable character][] and the method `getModifierState` returning `false`, and there is an [instrument][] to disable the [keyboard event][] and another [instrument][] to **remap** the [keyboard event][] so that it requires that the method `getModifierState` returns `true`. The [instruments][instrument] can be found from a [clearly labeled location][].
+
+```html
+<html>
+  <head>
+    <title>Passed Example 6</title>
+    <script src="/test-assets/ffbc54/shortcut.js"></script>
+    <script>
+      function openModal() {
+        document.getElementById('overlay').style.display = 'block'
+      }
+      function closeModal() {
+        document.getElementById('overlay').style.display = 'none'
+      }
+    </script>
```

Comment:
We may want to move these scripts to `shortcut.js` for the sake of keeping the inlined part of the example not too big.